ansible/controllerProvision.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- hosts: localhost
+  connection: local
+  tasks:
+  - name: Add controller to inventory
+    add_host:
+      name: masterController
+      ansible_host: "{{ controllerIP }}"
+      ansible_user: root
+      ansible_ssh_pass: contrail123
+      ansible_ssh_extra_args: '-o UserKnownHostsFile=/dev/null'
```

Comment:
Each variable added here, besides `ansible_host` can be added to `group_vars/controller`.